src/cli/init_command.zig

```diff
@@ -0,0 +1,80 @@
+const Command = @import("../cli.zig").Command;
+const std = @import("std");
+
+pub const InitCommand = struct {
+    fn getUserInput(alloc: std.mem.Allocator, default: []const u8) []const u8 {
+        const stdin = std.io.getStdIn().reader();
+
+        const buffer = stdin.readUntilDelimiterOrEofAlloc(alloc, '\n', 100) catch {
+            return default;
+        };
+    
+        const buf = buffer orelse "";
+        if (buf.len == 0) {
+            return default;
+        }  
+        else {
+            return buf;
+        }
+    }
+
+    pub fn exec(alloc: std.mem.Allocator) !void {
+        const stdout = std.io.getStdOut().writer();
+
+        // get package info from user
+        try stdout.print("package name: (project)", .{});
+        const project_name = getUserInput(alloc, "project");
+
+        try stdout.print("description: ", .{});
+        const desc = getUserInput(alloc, "");
+
+        try stdout.print("entry point: (index.ts)", .{});
+        const entry_point = getUserInput(alloc, "index.ts");
+
+        try stdout.print("author: ", .{});
+        const author = getUserInput(alloc, "");
+
+        try stdout.print("license: (ISC)", .{});
+        const license = getUserInput(alloc, "ISC");
+
+        const fs = std.fs;
+        const cwd = fs.cwd();
+
+        const package_json = cwd.createFile("package.json", .{ .exclusive = true }) catch |err| {
```

Comment:
Use `try` here so that bun's default error handling will take care of it
```suggestion
        const package_json = try cwd.createFile("package.json", .{ .exclusive = true });
```